### PR TITLE
Fix dimension mismatch in convert concat

### PR DESCRIPTION
**Pull request by RanHomri**

### CODE REVIEW

File: onnx2kerastl/utils.py [hunk 1]
```diff
@@ -131,3 +131,34 @@ def check_torch_keras_error(model, k_model, input_np, epsilon=1e-5, change_order
...
+
+    if len(set(ranks)) == 1:
+        return tensor_list
+
+    unsqueezed_tensors = []
+    for tensor in tensor_list:
+        tensor_rank = tensor.shape.rank
+        if tensor_rank < max_rank:
+            diff_shape = (1,) * (max_rank - tensor_rank) + tensor.shape
+            unsqueezed_tensor = keras.backend.expand_dims(tensor, axis=0)
```
Comment (doronHarnoy) [inline onnx2kerastl/utils.py]: how do we know only expand_dims in axis=0 is necessary?

File: onnx2kerastl/utils.py [hunk 2]
```diff
@@ -155,8 +156,9 @@ def match_tensors_rank(tensor_list):
     for tensor in tensor_list:
         tensor_rank = tensor.shape.rank
         if tensor_rank < max_rank:
-            diff_shape = (1,) * (max_rank - tensor_rank) + tensor.shape
-            unsqueezed_tensor = keras.backend.expand_dims(tensor, axis=0)
+            diff_rank = max_rank - tensor_rank
+            diff_shape = [1] * diff_rank + list(tensor.shape)
+            unsqueezed_tensor = tf.broadcast_to(tensor, diff_shape)
```
[the remark this answers is not in the document]
Comment (RanHomri) [reply onnx2kerastl/utils.py:161]: I will look into that
